src/_scss/pages/search/timePeriodFilter.scss

```diff
@@ -55,6 +57,42 @@
             @include usa-input;
             display:inline;
         }
+        .date-range-wrapper {
+
+            .errorMessage {
```

Comment:
This style is not applied because `errorMessage` is not a child class of `date-range-wrapper`. Also let's start using `dashed-names` instead of camelCase.